bounties/npm/mongo-parse/1/README.md

```diff
@@ -0,0 +1,3 @@
+# Overview
+
+`mongo-parse` is a parser for mongo db queries and projections, this package is vulnerable to Arbitrary Code Injection due to unsafe use of the `eval()` function. Node.js provides the `eval()` function by default, and is used to translate strings into Javascript code. An attacker can craft a malicious payload to inject arbitrary commands.
```

Comment:
The `eval` problem has been solved with a PR marged after the issue was created, however the fix introduced another similar issue because the `query` was formatted inside the `new Function()` snippet, making still possible `code injection`.

I suggest `updating` the description with the new case and point out this comment: https://github.com/fresheneesz/mongo-parse/issues/7#issuecomment-207829467, in order to make Huntrs able to understand what exactly wants the maintainer :smile:

Cheers,
Mik

Reply:
Thanks for the input @Mik317, the comment has been added as a reference and considered a valid bounty.

Cheers!